src/main/java/com/github/vkorobkov/jfixtures/sql/SqlBase.java

```diff
@@ -51,15 +52,23 @@ default void addCustomSql(Appender appender, CustomSql customSql) {
 
     default String escapeTableOrColumn(String name) {
         return Arrays
-            .stream(name.split("\\."))
-            .map(this::escapeTableOrColumnPart)
-            .collect(Collectors.joining("."));
+                .stream(name.split("\\."))
+                .map(this::escapeTableOrColumnPart)
+                .collect(Collectors.joining("."));
     }
 
     default String escapeValue(FixtureValue value) {
         String str = value.toString();
         return value.getType() == ValueType.TEXT ? SqlUtil.escapeString(str) : str;
     }
 
+    default String uppercaseNull(String value) {
+        String uppercasedNull = "NULL";
+        if (value == null || value.equalsIgnoreCase(uppercasedNull) || value.equals("~")) {
```

Comment:
This `value.equals("~")` looks like a bug - if the text is `~` we should have `'~'` in the SQL.

When snake yaml reads `~`  value it returns `null`, not a string with `~` literal

Reply:
This check `value.equalsIgnoreCase(uppercasedNull)` also seems like a bug:

if I define a property:
```yaml
name: 'Null'
```
I should have `'Null'` in SQL as well, because it is a string